### PR TITLE
Removes obsolete obj_flags flag

### PR DESCRIPTION
**Pull request by ChungusGamer666**
## About The Pull Request

This flag is literally only used in two objects in the game and seemingly does nothing

## Why It's Good For The Game

Removes obsolete code

## Changelog

no

### CODE REVIEW

**Comment by ChungusGamer666** [comment]
Integration test fail is not my problem

**Comment by ZephyrTFA** [comment]
let gh actions cook

File: code/game/objects/objs.dm [hunk 1]
```diff
@@ -123,32 +123,34 @@ GLOBAL_LIST_EMPTY(objects_by_id_tag)
 		return null
 
 /obj/proc/updateUsrDialog()
-	if((obj_flags & IN_USE) && !(obj_flags & USES_TGUI))
```
Comment (optimumtact) [inline code/game/objects/objs.dm:126]: it is clearly used, you can't just remove it without understanding why it exists.

Comment (ChungusGamer666) [reply code/game/objects/objs.dm:126]: "This flag is literally only used in two objects in the game and seemingly does nothing"
A bunch of objects that use TGUI, do not use this flag. Either this flag is obsolete, or there is something seriously wrong with most objects that use TGUI.

Comment (ChungusGamer666) [reply code/game/objects/objs.dm:126]: I am pretty sure it's the former, in fact.

**Comment by ChungusGamer666** [comment]
so?

**Pull request merged**